### PR TITLE
Fix #123: duplicated console log in the arbiter verify mode

### LINKED ISSUE #123: Arbiter console log are duplicated
In configuration verify mode (-V), the Alignak arbiter has duplicated console log once the arbiter module is loaded (see https://github.com/Alignak-monitoring/alignak/issues/726).

This because of a bad configuration of the module logger which declares a console handler...

### PR DESCRIPTION
**Pull request by mohierf**

### CODE REVIEW

File: alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_arbiter.cfg [hunk 1]
```diff
@@ -3,6 +3,9 @@ define module {
     module_types            configuration
     python_name             alignak_module_backend.arbiter
 
+    # Module log level
```
Comment (ddurieux) [inline alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_arbiter.cfg]: Add in comment the values possible please

File: alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_arbiter.cfg [hunk 2]
```diff
@@ -3,6 +3,9 @@ define module {
...
+    log_level               DEBUG
```
Comment (ddurieux) [inline alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_arbiter.cfg]: It's better to have INFO per default ;)

File: alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_broker.cfg [hunk 3]
```diff
@@ -3,6 +3,9 @@ define module {
     module_types            livestate, action
     python_name             alignak_module_backend.broker
 
+    # Module log level
```
Comment (ddurieux) [inline alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_broker.cfg]: Idem

File: alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_scheduler.cfg [hunk 4]
```diff
@@ -3,6 +3,9 @@ define module {
     module_types            retention
     python_name             alignak_module_backend.scheduler
 
+    # Module log level
```
Comment (ddurieux) [inline alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_scheduler.cfg]: idem

**Review by ddurieux — CHANGES_REQUESTED** [review]

File: alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_arbiter.cfg (hunk 1 above)
Comment (mohierf) [reply alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_arbiter.cfg]: Done

**Review by ddurieux — APPROVED** [review]

**Comment by ddurieux** [comment]
A little squash before I merge

**Comment by mohierf** [comment]
Why squashing? 3 commits for 3 independent modifications ... please merge as is.

**Pull request merged**